pkg/subctl/cmd/root.go

```diff
@@ -60,7 +61,9 @@ func init() {
 
 func addKubeconfigFlag(cmd *cobra.Command) {
 	cmd.PersistentFlags().StringVar(&kubeConfig, "kubeconfig", "", "absolute path(s) to the kubeconfig file(s)")
-	cmd.PersistentFlags().StringVar(&kubeContext, "kubecontext", "", "kubeconfig context to use")
+	cmd.PersistentFlags().StringVar(&kubeContext, "kubecontext", "", "DEPRECATED: use kubecontexts")
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&kubeContext, "kubecontext", "", "kubeconfig context to use")
	exitOnError("deprecation error", cmd.Flags().MarkDeprecated("kubecontext", "please use --kubecontexts instead"))
```

Reply:
We can't do this right now coz it will break lots of existing scripts. `kubecontext` should be allowed for now but let the user know that it has been deprecated and should use the new `kubecontexts`